src/extensions/surveys/surveys-utils.tsx

```diff
@@ -135,8 +135,7 @@ export const style = (appearance: SurveyAppearance | null) => {
           }
           .description {
               font-size: 13px;
-              margin-top: 5px;
-              opacity: .60;
```

Comment:
iirc this makes photos dimmer, like in this one -> https://us.posthog.com/project/2/surveys/018d3daf-6594-0000-4279-c8bf22b7f6ed -> but indeed doesn't make much sense to me, and people should be able to edit their css to make images lighter if they want to.

Reply:
and the preview doesn't have the opacity set either 🤦